sopht_mpi/numeric/eulerian_grid_ops/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""MPI-supported eulerian grid operations."""
+from .poisson_solver_2d import *
+from .stencil_ops_2d import *
```

Comment:
wild card star imports are bad in general (https://rules.sonarsource.com/python/RSPEC-2208), lets's have explicit imports, see `sopht-backend` for examples.

Reply:
good catch, actually I realized star imports, in this case, should be fine since we only have sub-modules in this folder, and we want to promote all internal features to the `eulerian_grid_ops` namespace. You can revert back here if you wish, apologies for the inconvenience, I'm fine with the current version as well.

Reply:
That's fine. I think its good that we do it explicit this way. So when we write new kernels we will just add it to the `__init__.py` and it is less prone to running into issues that is mentioned in the link you attached earlier. Thanks!